sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateSequencingParametersDao.java

```diff
@@ -46,7 +45,7 @@ public SequencingParameters getSequencingParameters(Long id) throws IOException
     if (id == null) {
       return null;
     }
-    return (SequencingParameters) currentSession().get(SequencingParametersImpl.class, id);
+    return (SequencingParameters) currentSession().get(SequencingParameters.class, id);
   }
```

Comment:
Please make changes to `getSequencingParameters()` as well; it's still looking for the Impl.